IDEAS/Examples/TwinHouses/BaseClasses/Ventilation/Vent_TTH_pressureNetwork_AirPerfectGass.mo

```diff
@@ -0,0 +1,92 @@
+within IDEAS.Examples.TwinHouses.BaseClasses.Ventilation;
```

Comment:
@kldjonge can you add documentation in the information section about why this class is needed? It sounds like you need a different Medium but you could have achieved that simply by redeclaring the Medium in the original model?